cypress/support/fragments/data_import/mapping_profiles/newFieldMappingProfile.js

```diff
@@ -242,7 +242,7 @@ export default {
 
     cy.do(Select({ name: statisticalCodeFieldName }).choose(actions.addTheseToExisting));
     cy.do(Button('Add statistical code').click());
-    cy.do(TextField('Statistical code').fillIn(name));
+    cy.do(TextField('Statistical code').fillIn(`"${name}"`));
```

Comment:
you can use just

`cy.do(TextField('Statistical code').fillIn(name));`

Reply:
I need double quotes when I'm creating mapping profile, but I don't need them when I'm searching item in Inventory by statistical code. That's why I use this constriction in creating function